### PR TITLE
Add point accuracy & define optional point feature properties

### PR DESCRIPTION
**Pull request by ezheidtmann**
Addresses #59 and #51. 

I used `horizontalAccuracy` because it's a good match to `verticalAccuracy`, but I don't feel strongly.

### CODE REVIEW

**Comment by ezheidtmann** [comment]
Thanks. Rebased.

File: provider/README.md [hunk 1]
```diff
@@ -120,17 +120,42 @@ Data: `{ "trips": [] }`, an array of objects with the following structure
...
 
+<<<<<<< HEAD
 To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route, and a timestamp.
+||||||| merged common ancestors
+To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route, and a timestamp. 
+=======
+To represent a route, MDS `provider` APIs should create a GeoJSON Feature Collection, which includes every observed point in the route. Each point is expressed as a GeoJSON `Feature` with a `Point` geometry.
+
+Route point features must include at least a UTC timestamp and accuracy radius in the `properties` key. Optional fields may be omitted or use the `null` value.
+>>>>>>> Add point accuracy & define optional point feature properties
```
Comment (thekaveman) [inline provider/README.md]: It looks like you rebased, but didn't resolve the merge conflicts?

Comment (ezheidtmann) [reply provider/README.md]: Ha! Looks like I committed before I saved the resolution. Pushed again.

**Comment by hunterowens** [comment]
sorry for the close - had some branch errors that required recreating master and that closed this PR.

**Comment by ezheidtmann** [comment]
What changes are needed now? Another rebase?

**Comment by karcass** [comment]
Is this PR still relevant?  @thekaveman?

**Comment by schnuerle** [comment]
Being addressed by #653. Please review that PR and leave comments there if interested.

**Pull request closed without merging**